cognite/powerops/client/data_classes/shop_results_compare.py

```diff
@@ -38,9 +47,221 @@ def plot_time_series(
         ax.legend()
         plt.show()
 
-    def yaml_difference(
+    def yaml_deep_diff(
         self,
-        yaml_1: ShopYamlFile,
-        yaml_2: ShopYamlFile,
+        post_run_yaml_1: ShopYamlFile,
+        post_run_yaml_2: ShopYamlFile,
+        **deepdiff_kwargs,
```

Comment:
Is this argument necessary to expose to the user? I think we should not expose it. 
```suggestion
```